x-pack/solutions/observability/plugins/metrics_data_access/server/lib/metrics/query.ts

```diff
@@ -6,8 +6,8 @@
  */
 
 import { decodeOrThrow } from '@kbn/io-ts-utils';
-import type { MetricsAPIResponse, MetricsAPIRequest } from '@kbn/metrics-data-access-plugin/common';
-import { TIMESTAMP_FIELD } from '../../../common/constants';
+const TIMESTAMP_FIELD = '@timestamp';
+import type { MetricsAPIRequest, MetricsAPIResponse } from '../../../common/http_api/metrics_api';
```

Comment:
Don't we have this field defined on a shared constant file? And do you really want to place it between imports?